python/ir/bindings.cpp

```diff
@@ -0,0 +1,133 @@
+#include <pybind11/functional.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "reffine/base/type.h"
+#include "reffine/builder/reffiner.h"
+#include "reffine/ir/expr.h"
+#include "reffine/ir/loop.h"
+#include "reffine/ir/op.h"
+#include "reffine/ir/stmt.h"
+#include "reffine/pass/printer.h"
+
+using namespace std;
+using namespace reffine;
+using namespace reffine::reffiner;
+
+namespace py = pybind11;
+
+void print_IR(shared_ptr<Func> fn)
```

Comment:
instead of this, write a function called to_string() that returns the serialized string of the function object.

Reply:
done.